release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -71,6 +71,16 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 
 {product-title} 4.11 introduces the `nvme-cli` package that provides an interface for communicating with NVMe disks using host software capabilities.
 
```

Comment:
This is a bit misleading; RHCOS has had support for using NVMe disks for a while.  The `nvme-cli` package is just a tool that can be used to manage NVMe devices.

From `rpm -qi nvme-cli`:

```
Summary     : NVMe management command line interface
Description : nvme-cli provides NVM-Express user space tooling for Linux.
```

Reply:
Thanks for catching this, fixed